3p/integration.js

```diff
@@ -456,14 +453,11 @@ export function validateParentOrigin(window, parentLocation) {
   // Currently only webkit and blink based browsers support
   // ancestorOrigins. In that case we proceed but mark the origin
   // as non-validated.
-  if (!ancestors || !ancestors.length) {
-    parentLocation.originValidated = false;
-    return;
-  }
-  user().assert(ancestors[0] == parentLocation.origin,
+  if (ancestors && ancestors.length) {
```

Comment:
I'd prefer if you kept the old logic, but returned and than have the assert as top level statement in the function. That makes it more clear that the assert is not the special case.